ImisRestApi/Escape/Payment/Data/GepgUtility.cs

```diff
@@ -26,7 +26,7 @@ public class GepgUtility
         string GepgPublicCertStorePath = string.Empty;
         string GepgPayCertStorePath = string.Empty;
 
-        string CertPass = "HPSS1234";
+        string CertPass = "passpass";//HPSS1234
```

Comment:
We need to add them as config

Reply:
Ok.

Reply:
Then the same thing with "GepgCertPass"

Reply:
I will add them to a config

Reply:
@delcroip changed, you can recheck this.